Updates to articles to conform to some documentation style. (#344)
swiftlang/swift-testing: Sources/Testing/Testing.docc/EnablingAndDisabling.md

- Note: A condition may be evaluated multiple times during testing.

## Disable a test

> If a test should be disabled unconditionally, you can use the

Reviewer: ```suggestion
If you need to disable a test unconditionally, use the
```

func sellsBurritos() async throws { ... }
```

## Conditionally enable or disable a test

> Sometimes, it makes sense to enable a test only if a condition is met. Consider

Reviewer: ```suggestion
Sometimes, it makes sense to enable a test only when a certain condition is met. Consider
```

> run. Otherwise, the first condition to fail will be noted as the reason the test
> was skipped.

Reviewer: ```suggestion
run. Otherwise, the first condition to fail is noted as the reason why the test
was skipped.
```

Reviewer 2: Well it hasn't happened yet—future tense to be avoided regardless?
